### ISSUE TITLE
Is the database supposed to be regularly purged?

### ISSUE THREAD

**Issue by blindreaper** [AHAAAAAAA/PokemonGo-Map#3341]
Just clicked full stats then All locations and with 17,000 pidgeys, my computer came to a halt.

**Comment by Devastatin** [comment]
can you zip the DB and post it here? I'd like to review the layout to see if streamlined database design was followed.

**Comment by blindreaper** [comment]
Seems the problem is all the icons for the pokemon have to load (>500 and it's slow with my specs).

**Comment by blindreaper** [comment]
Here's the dabase anyway if you want it, not sure how up to date it was because i converted to mysql. If it matters, i imported this into mariadb but i think the problem is the icons.

[pogom.zip](https://github.com/AHAAAAAAA/PokemonGo-Map/files/399733/pogom.zip)

**Comment by antiroot** [comment]
I agree with the issue that the map markers are causing a huge strain client side. Loading a heatmap for something that spawns very often with a few days worth of data causes my browser to become unresponsive. An option to toggle markers on and off would be great

**Comment by bambusbo** [comment]
How do you see all stats? i can only see the live stats and i have a mysql database connected

**Comment by Devastatin** [comment]
Bambushbo, im using the dev build, don't know what master looks like: http://localhost:5000/stats

I used sqlitebrowser.org to review the data.
select * from pokemon where pokemon_id ='1'; returns 258 rows almost instantly.  Maybe combining items that are so close in lat and long that only one marker needs to be used it the answer. I see that its already in place for exact spawn points. 
![capture](https://cloud.githubusercontent.com/assets/20668837/17374864/8f4f02c0-597c-11e6-9e89-1db4970a7f4a.PNG)
 but ones mere feet from each other spawns a new instance of icons. 
![image](https://cloud.githubusercontent.com/assets/20668837/17374904/c190f108-597c-11e6-8159-f74ad5a08ade.png)
